library/adodb/adodb-active-record.inc.php

```diff
@@ -1124,7 +1124,7 @@ function adodb_GetActiveRecordsClass(&$db, $class, $table,$whereOrderBy,$bindarr
 	// arrRef will be the structure that knows about our objects.
 	// It is an associative array.
 	// We will, however, return arr, preserving regular 0.. order so that
-	// obj[0] can be used by bootstrap developpers.
```

Comment:
Ha-ha!  This was changed by someone that did a huge SED on the whole code base replacing "app" with "bootstrap".  Though I caught all those a long time ago.